config/ompi_setup_prrte.m4

```diff
@@ -268,13 +273,15 @@ AC_DEFUN([_OMPI_SETUP_PRRTE_EXTERNAL], [
     AS_IF([test "$setup_prrte_external_happy" = "yes"],
           [AS_IF([test -n "$with_prrte"],
                  [PRTE_PATH="${with_prrte}/bin/prte"
-                  AS_IF([test ! -r ${PRTE_PATH}], [AC_MSG_ERROR([Could not find prte binary at $PRTE_PATH])])],
+                  AS_IF([test ! -r ${PRTE_PATH}],
+                        [AC_MSG_ERROR([Could not find prte binary at $PRTE_PATH])],
+                        [PRTE_PATH="${with_prrte}/bin"])],
 		 [PRTE_PATH=""
                   OPAL_WHICH([prte], [PRTE_PATH])
                   AS_IF([tets -z "$PRTE_PATH"],
                         [AC_MSG_WARN([Could not find prte in PATH])
-                         setup_prrte_external_happy=no])])])
-
+                         setup_prrte_external_happy=no],
+                        [PRTE_PATH="`echo $PRTE_PATH | sed -e 's/\/prte//'`"])])])
```

Comment:
Ugh, do we use `sed` elsewhere in configury?  I assume all Linuxes have `sed`.

Reply:
yeah - it is used elsewhere (example: opal_setup_wrappers.m4).

Reply:
yes; without sed, we would have much sadness....